xsql/parser.go

```diff
@@ -901,78 +911,124 @@ func validateStream(stmt *StreamStmt) error {
 	default:
 		return fmt.Errorf("option 'format=%s' is invalid", f)
 	}
+
+	if stmt.StreamType == TypeTable {
+		if t, ok := stmt.Options["TYPE"]; ok {
+			if strings.ToLower(t) != "file" {
```

Comment:
Would it be better to validate in the runtime instead of lexical phase? It could be probably complex to make this extensible?

Reply:
The same as above. Extensible to be supported later